probe-rs/src/architecture/arm/memory/adi_v5_memory_interface.rs

```diff
@@ -146,7 +146,7 @@ where
                 return Ok(());
             }
         }
-        Err(Error::Probe(DebugProbeError::Timeout))
+        Err(Error::Probe(DebugProbeError::Timeout("core reg transfer")))
```

Comment:
I would prefer to add a proper type for this, since it's not really a `DebugProbeError` anyway.

Reply:
Hmm I will try to do this. But I fear that we do not have access to `probe_rs::Error` on all levels this is used. I will try.